pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,233 @@
+
+Narzędzia nie mogą automatycznie sprawdzać wszystkich aspektów dostępności. Wymagany jest ludzki osąd. Czasami narzędzia oceny mogą dawać fałszywe lub wprowadzające w błąd wyniki. Narzędzia oceny dostępności stron internetowych nie mogą jednoznacznie *ocenić* dostępności, mogą jedynie w tym *pomóc*.
+
+
+## Własności narzędzi oceny
+{:#features}
+
+Narzędzia oceny dostępności cyfrowej są skierowane do różnych odbiorców. Korzystają z nich projektanci, programiści, nietechniczni autorzy treści, testerzy jakości, a czasem także końcowi użytkownicy. Narzędzia oferują różne możliwości i funkcje, które użytkownicy mogą porównać i ocenić, aby dobrać narzędzia pod kątem swoich konkretnych potrzeb.
+
+
+### Główne zastosowania
+
+Poniższe własności opisują narzędzia znajdujące się na [[Liście narzędzi oceny dostępności cyfrowej]](/test-evaluate/tools/list/). Można ich używać jako filtrów, aby zmniejszyć liczbę wyświetlanych narzędzi.
+
+**Cel**
+: Czy szukasz narzędzia, które wykona zautomatyzowany test, czy też narzędzie powinno pomóc Ci w ręcznym testowaniu lub symulacji doświadczenia użytkownika.
+
+**Oceniany produkt**
+: Zależnie od rodzaju produktu, który oceniasz (np. strona internetowa, dokument, aplikacja), możesz potrzebować konkretnego narzędzia, które będzie Ci najbardziej pomocne. Niektóre narzędzia są przystosowane do oceny stron internetowych, inne do dokumentów, aplikacji komputerowych, aplikacji mobilnych itp.
+
+**Płatne lub bezpłatne**
+: Narzędzia oceny są dostępne w ramach różnych typów licencji, takich jak licencja Open Source, komercyjna, korporacyjna itp. Niektóre narzędzia są płatne, inne darmowe.
+
+**Typ narzędzia**
+: Narzędziami oceny mogą być wtyczki (rozszerzenia) do narzędzi autorskich i przeglądarek internetowych, narzędzia wiersza poleceń, aplikacje komputerowe lub mobilne oraz usługi internetowe. W zależności od procesu projektowania i tworzenia można łączyć różne narzędzia.
+
+**Standardy**
+: Różne organizacje i rządy mogą wymagać zgodności z różnymi standardami dostępności, dlatego różne narzędzia obsługują te standardy. Wytyczne W3C dla dostępności treści internetowych (WCAG) są uznawane na arenie międzynarodowej za standard dostępności stron internetowych i obsługiwane przez większość narzędzi.
+
+**Typ pliku**
+: Większość narzędzi oceny sprawdza dostępność treści HTML. Niektóre sprawdzają inne technologie internetowe, takie jak EPUB, WAI-ARIA, CSS, SVG i PDF.
+
+**Zakres**
+: Zakres tego, co narzędzie oceny może automatycznie testować, różni się w zależności od narzędzia. Niektóre narzędzia sprawdzają pojedynczą stronę, inne natomiast sprawdzają całe grupy powiązanych stron. Niektóre mogą również uzyskać dostęp do treści chronionych hasłem.
+
+**System operacyjny**
+: Niektóre narzędzia działają tylko w określonych systemach operacyjnych, takich jak Windows, MacOS, Linux, Android, iOS.
+
+**Przeglądarki z wtyczkami**
```

Comment:
@zwiastunsw "Browser for plugin" means that this information is generally provided for "Browser plugins".

Does the translation reflects this?

Reply:
Ok, it will be better: Wtyczki przeglądarek